NEWS.md

```diff
@@ -1,3 +1,41 @@
+### v8.11.0 (2022-05-11)
+
+* Log forwarding from APM agent is now possible.
```

Comment:
true but that's not what this feature was completely.  I'd say perhaps `Application Logging in agent for winston now available`. Would like to get @coreyarnold feedback here

Reply:
might want to call out it can do log forwarding, adding log metrics and local log decoration